src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -94,7 +94,7 @@ const ArtistSeriesHeader: React.FC<ArtistSeriesHeaderProps> = ({
           </Text>
 
           {descriptionFormatted && (
-            <HTML variant="text">
+            <HTML variant="sm">
```

Comment:
Random thing I noticed. (We'll have to circle back here to refine the typography in this page's context bar as well.)